docs/source/release_notes/pre_release_epics.md

```diff
@@ -0,0 +1,91 @@
+
+
+## Cedar
+18 Nov 2018
+https://github.com/nucypher/nucypher/pull/490
+
+*Named as a homophone for "seeder", as this epic introduced our seeder contract and concepts.*
+
+* **Associated homecoming**: Prague / Devcon4
+
+The team hacked and slashed to prepare our first devnet for several "off Devcon" events, at which we presented demos using our first 10 public federated nodes.
+
+Finally coming to initial fruition around a table at a small restaurant in Prague (w/ @kprasch, @jMyles, @mswilkison, and @tuxxy frantically pecking), the devnet was demoed just an hour later.
+
+Following our first devnet demo, Cedar brought all of the changes that we launched into a more sane domain, adding tests for launch logic, docstrings, and logging statements throughout the codebase.  We also began reporting log emissions to Sentry.
+
+A lot of automation for deployment also landed in Cedar, including Ansible playbooks, our first status page, and the "nickname" scheme, in which a pronounceable name is derived from a node's checksum address.
+
+
+## Alpha
+
+*Although not associated with an epic branch, we conducted an Alpha Distribution release on Jan 29, 2019*
+
+Our Alpha release including a metric bajillion bugfixes, along with documentation, testing, tightened-up CI configuration, a new logo, and our "heartbeat" demo.
+
+
+## Carrot Juice
+15 Feb 2019
+https://github.com/nucypher/nucypher/pull/760
+
+*Named because jMyles had had enough coffee and needed a change.*
+
+* **Associated homecoming**: Denver / ETHDenver
+
+Focused on demo robustness and introducing relatively few features, Carrot Juice brought stability and sanity to some of the cutting-edge features developed up until then.
+
+Carrot Juice was the introduction of dual-use interfaces, meant both for live demos and for use by the public at large.
+
+Featuring 81 commits in the 24 hours leading up to the ETHDenver demo, Carrot Juice introduced Moe, our first "chaotic" character, and "MiddlewareClientWithConnectionProblems", our first network component focused entirely on testing how the network handles downtime.
+
+Carrot Juice was the basis for the ETHDenver demo:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/om0tew-Z4gE" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+
+## Vodka
+25 Feb 2019
+https://github.com/nucypher/nucypher/pull/507
+
+Our longest running epic - open for an entire season - Vodka offered a major critique to a several cryptoeconomic norms, most notably offering a zero-knowledge verification of re-encryption correctness.
+
+Vodka introduced a mutual commitment from Bob and Ursula, who now sign Alice's checksum address.  This represents the first time that Bob and Ursula have shared a common political view of the network, which is crucial for introducing accountability (including slashing) for our re-encryption service.
```

Comment:
```suggestion
Vodka introduced a mutual commitment from Bob and Ursula, who now sign the input and output of each re-encryption request.  This represents the first time that Bob and Ursula have shared a common political view of the network, which is crucial for introducing accountability (including slashing) for our re-encryption service.
```

The Alice's checksum address is going to go away soon, and in any case, it's not the important thing here: the mutual commitment to the input capsule and the resulting cfrag.